noxconfig.py

```diff
@@ -5,39 +5,60 @@
 from typing import Iterable
 
 from exasol.toolbox.nox.plugin import hookimpl
+from nox import Session
 
+DEFAULT_PORT = 8563
 DEFAULT_DB_VERSION = "8.31.0"
+CONTAINER_SUFFIX = "test"
+CONTAINER_NAME = f"db_container_{CONTAINER_SUFFIX}"
+
+
+def start_test_db(
+    session: Session, port: int = DEFAULT_PORT, db_version: str = DEFAULT_DB_VERSION
+) -> None:
+    # For Docker in a VM setup, refer to the ``doc/user_guide/developer_guide.rst``
+    session.run(
+        "itde",
+        "spawn-test-environment",
+        "--create-certificates",
+        "--environment-name",
+        CONTAINER_SUFFIX,
+        "--database-port-forward",
+        f"{port}",
+        "--bucketfs-port-forward",
+        "2580",
+        "--docker-db-image-version",
+        db_version,
+        "--db-mem-size",
+        "4GB",
+        external=True,
+    )
+    session.run(
+        "docker",
+        "cp",
+        f"{CONTAINER_NAME}:/certificates/rootCA.crt",
+        f"{Path(__file__).parent}/test/data/rootCA.crt",
+    )
```

Comment:
Maybe, move the copy cmd to a pytest fixture, which copies the certificate from the docker container into a temporary folder, which will be removed after the tests finished. just an idea....